onnx/version_converter.py

```diff
@@ -0,0 +1,44 @@
+# ATTENTION: The code in this file is highly EXPERIMENTAL.
+# Adventurous users should note that the APIs will probably change.
+
+"""onnx version converter
+
+This enables users to convert their models between different opsets.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import onnx
+import onnx.onnx_cpp2py_export.version_converter as C
+from onnx import ModelProto, OperatorSetIdProto
+from typing import Text, Sequence
+
+"""Apply the version conversion on the serialized ModelProto.
+
+Arguments:
+    input (ModelProto): model
+    initial_version (OperatorSetIdProto): initial opset version
+    target_version (OperatorSetIdProto): target opset version
+
+Return:
+    return (ModelProto) converted model
+
+Supported adapters:
+    --
+"""
+
+
+def convert_version(model, initial_version, target_version):  # type: (ModelProto, OperatorSetIdProto, OperatorSetIdProto) -> ModelProto
```

Comment:
Shouldn't the `initial_version` be taken from the model?

Reply:
The model may contain several opset_imports. I agree with that if no initial_version is specified, we just use the only opset_import in the model.

Reply:
After the discussion, let's stick to the opset_imports in model.

The new API should only contain model and target_version.